### DISCUSSION TITLE
Transmission downloads stalled with mikrotik router

### DISCUSSION THREAD

**Discussion by MasterCATZ** [transmission/transmission#8427] [Support]
### What is the issue?

Ubuntu 24.04
Transmission Version 
4.0.5 (a6fe2a64aa)

for some odd reason since switching over to a mikrotik router I can no longer get transmission to download / seed torrents

testing the ports it always says closed or status unknown

my qbittorrent / deluge clients still work just fine 

how ever I use Transmission for private trackers with peer discovery / DHT disabled 
![Screenshot from 2024-09-22 11-57-47](https://github.com/user-attachments/assets/ad38e1a9-c4de-4058-90b9-697ef298413e)

I can also install an old version of utorrent through wine and it worked just fine , 
instantly started downloading ubuntu iso  

idea's ?  also tried 4.0.6 but the trackers banned that version ... 

### Which application of Transmission?

None

**Comment by zivkovic** [comment]
You need to either port forward the port on the router, or enable UPNP(default is disabled). On how, do google for your router.

https://help.mikrotik.com/docs/display/ROS/UPnP

But I do recommend to simply port forward, UPNP is dangerous.

**Comment by MasterCATZ** [comment]
UPNP is enabled and all other programs are doing  NAT just fine, 

just not Transmission , and its not because of WireGuard either 
push comes to shove I will open up a dedicated port just for Transmission 

![Screenshot from 2024-09-24 18-03-01](https://github.com/user-attachments/assets/28baa3a2-5d98-4445-9440-fae85553dbdf)

**Comment by MasterCATZ** [comment]
and if I disable VPN / not use wireguard then upnp settings are added , but still no connections it gets peers and that is it  
same if I open port directly to it , it still thinks its closed , no firefall on the PC 

also ipv6 is in  use and it does not have acess to it either 

![Screenshot from 2024-09-24 19-38-41](https://github.com/user-attachments/assets/8ea4e837-40c9-4760-b121-73b5c6282e2a)
